### ISSUE TITLE
Monthly Budget Calculations are Incorrect

### ISSUE THREAD

**Issue by sraghun** [bbuchanan/los-issues#81]
The Monthly Budget Calculations are showing incorrect numbers.  I just now did a deal which had a monthly budget of $268 - $288 Monthly Payment for a period of 24 months at 17.99% APR when the amount financed was more than $18,000.   Obviously, this is wrong since the sum of all payments is less than $7,000.  Please see attached screenshot.

Also, the initial investment number seems to be wrong as well.
<img width="1710" alt="Screen Shot 2020-02-07 at 9 52 21 PM" src="https://user-images.githubusercontent.com/17339794/74078090-2cc24b00-49f4-11ea-9919-be4c4e26af4e.png">

**Comment by bbuchanan** [comment]
I agree the calculation is wrong. I was trying to see the difference(s) here between my work and Mike's spreadsheet, but it looks like Mike calculates and gets the same result, so I may not understand what some of these fields mean. Going to follow up w/ Mike now.

<img width="607" alt="Screen Shot 2020-02-10 at 13 06 44" src="https://user-images.githubusercontent.com/315135/74176706-36230180-4c06-11ea-95b5-bbeff4bf0411.png">

**Comment by bbuchanan** [comment]
Had a call w/ Mike. He's figured out the problem and is working on a solution in the spreadsheet. Will assign this to him for now so he can update it.

**Comment by humike** [comment]
Okay, I figured it out. 
We need to adjust the last bit of the formula to exclude Paytype. Basically, it's passing through 24 years in the spreadsheet, not months. 

((APR / PAYTYPE)*AMOUNT_TO_FINANCE / (1-((1 + (APR / PAY_TYPE))^(-TERM)))

That will calculate the appropriate payment associated with the loan. 

To convert to monthly, we need to multiply the result with the following:
IF(num_of_payments > 1, pay type / 12 * payment, payment)

I confirmed with Loan table that calc is working.
[PAYMENT FORMULA - updated.xlsx](https://github.com/bbuchanan/los-issues/files/4182527/PAYMENT.FORMULA.-.updated.xlsx)
